Source/WebCore/Modules/WebGPU/GPUPresentationContext.cpp

```diff
@@ -41,10 +41,39 @@ void GPUPresentationContext::unconfigure()
     m_backing->unconfigure();
 }
 
-GPUTexture* GPUPresentationContext::getCurrentTexture()
+void GPUPresentationContext::clearCurrentTextureAndView()
 {
-    // FIXME: https://bugs.webkit.org/show_bug.cgi?id=250958 Figure out how the lifetime of these objects should behave.
-    return nullptr;
+    m_currentTexture = nullptr;
+    m_currentTextureView = nullptr;
+}
+
+void GPUPresentationContext::ensureCurrentTextureAndView()
+{
+    ASSERT(static_cast<bool>(m_currentTexture) == static_cast<bool>(m_currentTextureView));
+
+    if (m_currentTexture && m_currentTextureView)
+        return;
+
+    m_currentTexture = GPUTexture::create(m_backing->getCurrentTexture()).ptr();
+    m_currentTextureView = GPUTextureView::create(m_backing->getCurrentTextureView()).ptr();
```

Comment:
I don't see this used anywhere by the web API. Won't this result in creating an extra texture view every frame?